src/CommunityToolkit.Maui/Behaviors/UserStoppedTypingBehavior.shared.cs

```diff
@@ -121,38 +121,40 @@ protected virtual void Dispose(bool disposing)
 		}
 	}
 
-	void OnTextPropertyChanged()
+	async void OnTextPropertyChanged()
 	{
-		if (tokenSource != null)
+		if (tokenSource is not null)
 		{
 			tokenSource.Cancel();
 			tokenSource.Dispose();
 		}
+
 		tokenSource = new CancellationTokenSource();
 
-		Task.Delay(StoppedTypingTimeThreshold, tokenSource.Token)
-			.ContinueWith(task =>
+		var task = Task.Delay(StoppedTypingTimeThreshold, tokenSource.Token);
+
+		try
+		{
+			await task.ConfigureAwait(ConfigureAwaitOptions.SuppressThrowing | ConfigureAwaitOptions.ContinueOnCapturedContext);
```

Comment:
Adding `ConfigureAwaitOptions.SuppressThrowing` changes the original behavior of this method.

The current/previous implementation throws an exception when `Task.Delay` is canceled via `tokenSource.Token`. 

This new implementation suppresses that exception. 

Honest question - is our intention to throw an `OperationCanceledException` when the `CancellationToken` is cancelled, or do we want to suppress it? I haven't yet done my due diligence and dug into the code surrounding this to know which is the correct answer, but I wanted to highlight this small "breaking change".